assets/section-main-product.css

```diff
@@ -262,6 +262,7 @@ a.product__text {
 
 .product .price {
   align-items: flex-start;
+  gap: 1rem;
```

Comment:
This gap is to fix [this issue](https://screenshot.click/21-57-wos8h-o5r9k.png) which only happens at specific viewport widths for on sale items with unit pricing. I chose 1rem so it can take the place of the existing horizontal 1rem margin between the sale price and the sale badge (when displayed next to each on larger breakpoints).

Reply:
I like this clean approach! But some versions of the latest iOS Safari 14 release (< 14.4) don't support this.

![Screen Shot 2021-08-17 at 2 12 08 PM](https://user-images.githubusercontent.com/3710548/129778666-4092e478-286c-4dac-9898-3368c27d84d6.png)

![Screen_Shot_2021-08-17_at_2_07_32_PM](https://user-images.githubusercontent.com/3710548/129778702-013e389c-2c2d-4230-b06c-fc2d374e40d9.jpg)